src/Phinx/Db/Adapter/AdapterInterface.php

```diff
@@ -48,6 +48,7 @@
     const PHINX_TYPE_INTEGER        = 'integer';
     const PHINX_TYPE_BIG_INTEGER    = 'biginteger';
     const PHINX_TYPE_FLOAT          = 'float';
+    const PHINX_TYPE_DOUBLE         = 'double';
```

Comment:
Adding a global type is typically not the most desirable way to add support for something. Correct way to add support for `double` specifically for Postgres would be to implement an option for `float` that triggers the standard type to be converted into a `double precision` by Postgres.